lib/src/specs/image/image_util.dart

```diff
@@ -45,4 +53,21 @@ class ImageUtility extends SpecUtility<ImageSpecAttribute> {
   DoubleUtility<ImageSpecAttribute> get height {
     return DoubleUtility((height) => _only(height: height));
   }
+
+  AlignmentUtility<ImageSpecAttribute> get alignment {
+    return AlignmentUtility((alignment) => _only(alignment: alignment));
+  }
+
+  RectUtility<ImageSpecAttribute> get centerSlice {
+    return RectUtility((rect) => _only(centerSlice: rect));
+  }
+
+  FilterQualityUtility<ImageSpecAttribute> get filterQuality {
+    return FilterQualityUtility(
+        (filterQuality) => _only(filterQuality: filterQuality));
+  }
+
+  BlendModeUtility<ImageSpecAttribute> get blendMode {
+    return BlendModeUtility((blendMode) => _only(blendMode: blendMode));
+  }
```

Comment:
Let's also add a `call()` method. One to keep in mind, though, is the call() method takes the actual Flutter core classes, not Dtos, if they exist.